app/src/coin.h

```diff
@@ -49,7 +49,7 @@ extern "C"
 #define APPVERSION_LINE2 ("v" APPVERSION)
 
 #define CRYPTO_BLOB_SKIP_BYTES 0
-#define COIN_DEFAULT_CHAINID "kava-3"
+#define TESTNET_CHAINID_PREFIX "kava-testnet"
```

Comment:
Generally, our testnets have names like `testnet-8000` or `testnet-9000`. Local development networks usually have a chain ID of `testing`.